linux_os/guide/services/kerberos/kerberos_disable_no_keytab/oval/shared.xml

```diff
@@ -0,0 +1,25 @@
+<def-group>
+  <definition class="compliance" id="kerberos_disable_no_keytab" version="1">
+    <metadata>
+      <title>Restrict Kerberos operation by removing keytab files</title>
+      <affected family="unix">
+        <platform>multi_platform_fedora</platform>
+        <platform>Red Hat Enterprise Linux 8</platform>
+        <platform>Oracle Linux 8</platform>
+      </affected>
+      <description>Check that there is no Kerberos keytab file present in /etc</description>
+    </metadata>
+    <criteria>
+      <criterion test_ref="test_kerberos_disable_no_keytab" negate="true"
```

Comment:
I think instead of using negation, you can set `check="all" check_existence="none_exist"` in the `file_test` element and remove the `negate` attribute. However, your solution works for me as well.